kernels/HausdorffKernel.cpp

```diff
@@ -86,12 +86,13 @@ int HausdorffKernel::execute()
     ColumnPointTable candTable;
     PointViewPtr candView = loadSet(m_candidateFile, candTable);
 
-    double hausdorff = Utils::computeHausdorff(srcView, candView);
+    std::pair<double, double> result = Utils::computeHausdorff(srcView, candView);
```

Comment:
Should we leave a Utils::computeHausdorff around that returns a double and make a new Utils::computeHausdorffPair method that does this? I'm unsure if there's any users of Utils::computeHausdorff though, so maybe it doesn't matter.

Reply:
I initially thought we'd only be impacting ourselves via the call from the kernel, but I suppose since it's publicly available via Utils, we should still preserve the original. I can make this change.